migrate/20250121_kubernetes_tags.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    run <<~SQL
```

Comment:
Just FYI, Sequel does not support multiple statements in a single `run` call.  This happens to work on the postgres adapter, but it can fail on other adapters.  It's not required, but its best if each statement has its own `run` method.

Reply:
Good to know, made it all.